Website/src/rendering/src/components/FeaturedSpeakers.tsx

```diff
@@ -1,71 +1,65 @@
 import Link from 'next/link';
-import Image from 'next/image';
 
-import speaker1 from '../../data/media/img/speakers/mary-asada.jpeg';
-import speaker2 from '../../data/media/img/speakers/martin-moore.jpeg';
-import speaker3 from '../../data/media/img/speakers/ed-jones.jpeg';
-import speaker4 from '../../data/media/img/speakers/sophia-taylor.jpeg';
-import speaker5 from '../../data/media/img/speakers/li-xiu-ying.jpeg';
+import { Text, Field, ImageField } from '@sitecore-jss/sitecore-jss-nextjs';
```

Comment:
Why is there an empty line between these 2 blocks of JS variable imports? Can it please be removed?